changes/change.go

```diff
@@ -0,0 +1,93 @@
+package changes
+
+import (
+	"errors"
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"strings"
+	"time"
+)
+
+// changeTypes maps valid Change Types to the header they are grouped under in CHANGELOGs.
+var changeHeaders = map[string]string{
+	"feature": "New Features",
+	"bugfix":  "Bug Fixes",
+}
+
+const changeTemplateSuffix = `
+# type may be one of "feature" or "bugfix".
+# multiple modules may be listed. A change metadata file will be created for each module.`
+
+type changeTemplate struct {
+	Modules     []string
+	Type        string
+	Description string
+}
+
+// Change represents a change to a single Go module.
+type Change struct {
+	ID            string // ID is a unique identifier for this Change
+	SchemaVersion string // SchemaVersion is the version of the library's types used to create this Change.
+	Module        string // Module is the Go module affected by this Change.
+	Type          string // Type indicates what category of Change was made. Type may be either "feature" or "bugfix".
+	Description   string // Description is a human readable description of this Change meant to be included in a CHANGELOG.
+}
+
+// NewChanges returns a Change slice containing a Change with the given type and description for each of the specified
+// modules.
+func NewChanges(modules []string, changeType, description string) ([]*Change, error) {
```

Comment:
Yeah, that's correct. I did it this way since it's easier to enter the shortened relative module path (like `aws/module`) for the developer. It might make sense to take the developer entered short module path and store it in a Change as the full module import path.